ArduPlane/Log.cpp

```diff
@@ -374,8 +374,8 @@ const struct LogStructure Plane::log_structure[] = {
 // @Field: CRt: climb rate
 // @Field: TMix: transition throttle mix value
 // @Field: Sscl: speed scalar for tailsitter control surfaces
-// @Field: Trn: Transistion state
-// @Field: Ast: Q assist active state
+// @Field: Trn: Transistion state: 0 AirspeedWait/1 Timer/2 Done, TailSitter: 0 FW Wait/1 VTOL Wait/ 2 Done
```

Comment:
Should be using `FieldBitmaskEnum` e.g. https://github.com/ardupilot/ardupilot/blob/pmaster/libraries/AP_Logger/LogStructure.h#L720

```suggestion
// @Field: Trn: Transition state: 0 AirspeedWait/1 Timer/2 Done, TailSitter: 0 FW Wait/1 VTOL Wait/ 2 Done
```

Reply:
The problem is its two enums, there is one for tailsitter and one for other quadplanes.

Reply:
and the suggested change is historical but fixed